packages/providers/mgt-mock-provider/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@microsoft/mgt-mock-provider",
-  "version": "4.3.0",
```

Comment:
@musale  why not have a global version variable/parameter in one config to update all these files? Much cleaner and less error prone in case someone forgets to bump the version across all these files

Reply:
The release please config json file cofigures how you can update the version for specific files. In our case, this is the config https://github.com/microsoftgraph/microsoft-graph-toolkit/blob/main/release-please-config.json and release please handles all version bumps.